concourse/scripts/import_bosh_ca.sh

```diff
@@ -3,4 +3,8 @@ set -eu
 
 echo "Adding bosh-CA to root certificates"
 tar -xzf bosh-CA/bosh-CA.tar.gz -C /usr/local/share/ca-certificates bosh-CA.crt
-update-ca-certificates
+
+UPDATE_OUTPUT=$(update-ca-certificates 2>&1)
+set +e
+printf "%s" "${UPDATE_OUTPUT}" | grep -i -v "skipping$"
```

Comment:
Is it worth specifying the string to match to be something closed to the error message? I don't like the idea that we'd also drop lines that happened to contained "skipping" in them. Also, I don't think we need to be case-insensitive either.

Something like this?:

`grep -v "^WARNING: ca-certificates\.crt does not contain exactly one certificate or CRL: skipping$"`

Reply:
Updated